harper-core/tests/text/tagged/Part-of-speech tagging.md

```diff
@@ -242,8 +242,8 @@
 # NSg    . V/C NSg/V  D/P NSg/V V/P D   NPl           V/P I/J     NPl       . C/P
 > example , once  you've seen  an  article such  as    ' the ' , perhaps the next    word  is a
 # NSg/V   . NSg/C W?     NSg/V D/P NSg/V   NSg/I NSg/R . D   . . NSg     D   NSg/J/P NSg/V VL D/P
-> noun  40 % of  the time  , an  adjective 40 % , and a   number  20 % . Knowing   this , a
-# NSg/V #  . V/P D   NSg/V . D/P NSg/V/J   #  . . V/C D/P NSg/V/J #  . . NSg/V/J/P I/D  . D/P
+> noun  40 % of  the time    , an  adjective 40 % , and a   number  20 % . Knowing   this , a
+# NSg/V #  . V/P D   NSg/V/J . D/P NSg/V/J   #  . . V/C D/P NSg/V/J #  . . NSg/V/J/P I/D  . D/P
```

Comment:
[Time](https://en.wiktionary.org/wiki/time#English) isn't an adjective. Similarly, the following words are also incorrectly tagged as adjectives now: year, hour, night, love, death, sight, earth, father, daughter, manner, purpose, swimming.

The adjective property assigned via `base_metadata` should probably be removed.

Reply:
Actually that's because many of those `-ly` words are not adverbs but adjectives so need to be fixed in the dictionary.

All from yearly to fatherly are adjectives. Swimmingly is an adverb though and swimming is a present participle and present participles are adjectives. The others I've never seen before.

Correction: They are adjectives but some are both adjective+adverb. The dictionary entries need to be fixed either way.

I also think the base POS property shouldn't be assigned for any 'affix' but it's all through the system for the other affixes. It is going to be needed for fixing the system though, which should check the base POS before adding the affix and assigning the derived POS.

I'll fix all the POSes in a new commit after cross-checking with dictionaries...

Reply:
> All from yearly to fatherly are adjectives.

That's not what I was saying. I was talking about year to father. No -ly. See the line of code I commented on. "time" is incorrectly tagged as an adjective.

Reply:
> > All from yearly to fatherly are adjectives.
> 
> That's not what I was saying. I was talking about year to father. No -ly. See the line of code I commented on. "time" is incorrectly tagged as an adjective.

Yes it was wrong for them to use `Y` so I've corrected them in the dictionary. I was only explaining why they had `Y` for Hunspell. We've gotta clean that kinda stuff up for Harper.